src/app.jsx

```diff
@@ -91,18 +99,25 @@ class Application extends React.Component {
                     this._asyncRequestContainers = null;
                     this.setState({containersMeta: reply.containers || []});
                     this.state.containersMeta.map((container) => {
-                        utils.varlinkCall(utils.PODMAN, "io.podman.InspectContainer", JSON.parse('{"name":"' + container.id + '"}'))
+                        utils.varlinkCall(utils.PODMAN, "io.podman.InspectContainer", {name: container.id})
+
                                 .then(reply => {
-                                    const temp_containers = this.state.containers;
+                                    let temp_containers = this.state.containers;
                                     temp_containers.push(JSON.parse(reply.container));
                                     this.setState({containers: temp_containers});
                                 })
-                                .catch(ex => console.error("Failed to do InspectImage call:", ex, JSON.stringify(ex)));
+                                .catch(ex => console.error("Failed to do InspectContainer call:", ex, JSON.stringify(ex)));
                     });
-                    this.state.containersMeta.map((container) => {
-                        utils.varlinkCall(utils.PODMAN, "io.podman.GetContainerStats", JSON.parse('{"name":"' + container.id + '"}'))
+                    this.state.containersMeta.filter((ele) => {
```

Comment:
Can you please document (in the constructor) what exactly `state.containers`, `state.containersMeta`, and `state.containersStats` are? an example how the state looks like would be helpful for following what's going on between these three. Also, it's not helpful for them to be lists if you have to modify them - I would expect these would all just be set from `utils.updateContainers()` (or `updateImages()` etc.) exclusively.